anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/ewma/EwmaAnomalyDetector.java

```diff
@@ -82,14 +82,14 @@ public AnomalyResult classify(MetricData metricData) {
         final AnomalyThresholds thresholds = new AnomalyThresholds(
                 this.mean + strongDelta,
                 this.mean + weakDelta,
-                this.mean - strongDelta,
-                this.mean - weakDelta);
+                this.mean - weakDelta, this.mean - strongDelta
```

Comment:
Change of indent seems a bit inconsistent.

Reply:
Yeah, the auto-formatter did that. It did that in multiple places, and I fixed some, but after a while I stopped as I didn't consider this blocking.

Reply:
OK just pushed a cleanup